test/factories/pets.rb

```diff
@@ -22,6 +22,10 @@
       adopter_applications { build_list(:adopter_application, 1, :successful_applicant, pet: instance) }
     end
 
+    trait :fostered do
```

Comment:
I think we should actually set up three foster traits, each differs based on the start and end dates. 
`completed_foster` = start and end dates are Time.current - X
`current_foster` = start date is Time.current - X. end date is Time.current + X
`future_foster` = start and end dates are Time.current + X

Does this make sense?

Reply:
Have implemented three different traits in pet factory and have used in test to see that only current pets are returned in fostered_pets dashboard